package.json

```diff
@@ -22,10 +22,10 @@
   "scripts": {
     "build": "rollup --environment NODE_ENV:'production' -c rollup.config.mjs && tsc --build tsconfig.declaration.json",
     "build:dev": "rollup --watch --environment NODE_ENV:'development' -c rollup.config.mjs",
-    "lint": "tslint ./lib/**/*.ts",
-    "lint:fix": "yarn lint -- --fix",
+    "lint": "eslint --ext .js,.ts,.tsx lib",
+    "lint:fix": "npm run lint -- --fix",
     "prettier": "prettier 'lib/**/*.ts' 'docs/**/*.md' --write",
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
Oddly this change resulted in the tests finishing _faster_. 🤷‍♂️

Reply:
Were you able to discover the underlying issue? Is it just that the tests are designed in such a way that they can't be done in parallel?

Reply:
I tried a few different things but could not figure it out. I thought that workers might have been getting held open due to our use of XHR (and so jsdom's implementation of it), but waiting on those requests in the tests didn't fix anything and made them take a lot longer.